records/new/mhdawson_bsky_social_post_3ls2faw4f2s24.json

```
@@ -0,0 +1,5 @@
+{
+  "action": "repost",
+  "account": "NODEJS_ORG",
+  "repostURL": "https://bsky.app/profile/mhdawson.bsky.social/post/3ls2faw4f2s24"
```

```suggestion
  "action": "post",
  "account": "NODEJS_ORG",
  "richText": "Recording from the #nodejs Node-api team meeting today in case you want to catch up https://www.youtube.com/watch?v=_eCD-T1DZsU"
```

@jasnell In case you didn't notice, this is how the suggestion is supposed to work in this repository.